src/app/stacktrace_win.h

```diff
@@ -262,6 +334,8 @@ const QString straceWin::getBacktrace()
     //logStream << "\n\nList of linked Modules:\n";
     //EnumModulesContext modulesContext(hProcess, logStream);
     //SymEnumerateModules64(hProcess, EnumModulesCB, (PVOID)&modulesContext);
+    SymCleanup(hProcess);
```

Comment:
@SeigneurSerpent:  If no other `SymInitialize` functions are being used where this code would affect it, then it would probably be best to comment it out like the lines above.  

If, on the other hand, `SymInitialize` functions are used somewhere else in the file, then you should probably leave it in.